plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/BaseKtlintCheckTask.kt

```diff
@@ -147,6 +150,12 @@ abstract class BaseKtlintCheckTask(
             enabledReports
                 .map { it.asArgument() }
                 .forEach { argsWriter.println(it) }
+            disabledRules
+                .get()
+                .fold("") { acc, rule -> "$acc,$rule" }
```

Comment:
I think you want `joinToString` or something here. You can pass a seperator.

Reply:
You are right! Totally forgot about it :man_facepalming: